elixir/lang/best-practice/deprecated-bnot-operator.yaml

```diff
@@ -0,0 +1,15 @@
+rules:
+  - id: deprecated_bnot_operator
+    message: >-
+      The bitwise operator (`^^^`) is already deprecated. Recommended to use `Bitwise.bnot/2` instead.
```

Comment:
```suggestion
      The bitwise operator (`^^^`) is already deprecated. Please use use `Bitwise.bnot($VAL)` instead.
```

Reply:
This already fixed in https://github.com/returntocorp/semgrep-rules/pull/3064/commits/4e9afa94685baeac65246f924fe5280986208b0a